src/main.cpp

```diff
@@ -42,7 +42,7 @@ int main(int argc, char* argv[])
 
         impactX->initData();
         impactX->initElements();
-        impactX->evolve( /* num_steps = */ 10);
+        impactX->evolve( /* num_steps = */ 1);
```

Comment:
In a follow-up PR, we could make this a user inputs option and pass no argument here (we can make `num_steps` a `std::optional<int>`).

Reply:
Yes, but we need to discuss how num_steps is used internally.  We may need to input a number of steps for each individual beamline element, as is done in IMPACT-Z.

Reply:
Sounds good, added to next meeting agenda :)